pkg/preflights/host-preflight.yaml

```diff
@@ -381,3 +435,339 @@ spec:
               message: 'P99 write latency for the disk at /var/lib/k0s/etcd is {{ "{{" }} .P99 {{ "}}" }}, which is better than the 10 ms requirement.'
           - fail:
               message: 'P99 write latency for the disk at /var/lib/k0s/etcd is {{ "{{" }} .P99 {{ "}}" }}, but it must be less than 10 ms. A higher-performance disk is required.'
+    - tcpPortStatus:
+        checkName: ETCD Internal Port Availability
+        collectorName: ETCD Internal Port
+        outcomes:
+          - fail:
+              when: "connection-refused"
+              message: Connection to 2379/TCP was refused. This may indicate that the connection is being redirected to somewhere else on your network (perhaps a proxy configuration).
```

Comment:
Yeah, we do that ourselves now. Maybe a more generic message like "ensure this port is available" would suffice if it's very unlikely this even occurs.

Reply:
After looking at this again I don't think even an invalid `NO_PROXY` configuration would cause this to happen. The collector operates at the TCP level. Are we OK with the "Please ensure this port is available" message ?

Reply:
Yeah, I can make a change